### PR TITLE
Update README with more install steps

### PR DESCRIPTION
**Pull request by azaslavsky**
I've gone through the README to setup my own dev environment, and document/clarified/fixed any typos or missing instructions I found along the way.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,40 +92,73 @@ You'll also need to create a private key for the service account (it should down
...
 
-NOTE: This steps will cover more aspects over time. For now it focuses on testing Slack/Github changes.
+NOTE: These steps will cover more aspects over time. For now it focuses on testing Slack/Github changes.
 
-1. Set up [Ngrok](https://ngrok.io/) to redirect calls to your localhost
-    - `ngrok config add-authtoken <Your Ngrok Auth Token>`
-    - `ngrok http 3000` --> Grab the URL ngrok gives you (e.g. `https://6a88fe29c5cc.ngrok.io`) and save it for step 6
-2. Create a new Slack workspace from the Slack app (e.g. `Sentry (testing)`). Do not use the Sentry workspace!
+1.  Set up [Ngrok](https://ngrok.io/) to redirect calls to your localhost.
+    - If you haven't already, sign up for an ngrok account and grab the auth token from [https://dashboard.ngrok.com/get-started/setup](the setup page).
```
Comment (chadwhitacre) [inline README.md:98]:
```suggestion
    - If you haven't already, sign up for an ngrok account and grab the auth token from [the setup page](https://dashboard.ngrok.com/get-started/setup).
```

File: README.md [hunk 2]
```diff
@@ -92,40 +92,73 @@ You'll also need to create a private key for the service account (it should down
...
+    - Some of the settings will need to be verified before they get saved
+    - This means that you will need to update your `.env` file with the settings from your Slack app
+    - Reload your server for the new env vars to apply and resend the verification payloads
+    - You will have to do this with multiple settings, thus, you will have to repeat reloading your server as you add new variables
+
+1.  Create a new GitHub repository at `<YOUR_GITHUB_USERNAME>/eng-pipes-dev`.
+1.  Create a [new GitHub App](https://github.com/settings/apps/new)
+    - Set the webhook to your ngrok tunnel with the GH route (e.g. `<NGROK_INSTANCE>/webhooks/github`)
+    - Place the secrets in your `.env` (see [Setup Secrets](#setup-secrets) below)
+    - Go to the `Permissions & events` sidebar menu entry of the GitHub app configuration, and grant maximum non-`Admin` access (`Read and write` where possible, `Read only` everywhere else) for every line in `Repository permissions`
```
Comment (chadwhitacre) [inline README.md:126]: That's one way to do it. :)

This isn't how we run in prod. Do we care? Should we at least say something about that?

Comment (hubertdeng123) [reply README.md:126]: A note would be nice. People using eng-pipes may not have access to the exact permissions and these may change in the future

File: README.md [hunk 3]
```diff
@@ -92,40 +92,73 @@ You'll also need to create a private key for the service account (it should down
...
+1.  Verify that the GitHub -> eng-pipes server pipeline works
+
+    - In your `eng-pipes-dev` GitHub repository, create a new issue
+    - You should see your localhost app response with a 200 status code:
+
+    ![success!](/docs/successful_github_event.png 'Successful GitHub webhook reception')
+
+Congratulations, you're all setup!
+
+### Quality-of-life tips
```
Comment (hubertdeng123) [inline README.md:149]: should we add something in here about the `DRY_RUN` variable? Since it gets rid of the bad auth requests to BigQuery

Comment (chadwhitacre) [reply README.md:149]: YEASSSSS

Comment (azaslavsky) [reply README.md:149]: Done.

File: README.md (hunk 2 above)
Comment (azaslavsky) [reply README.md:126]: Done.

File: README.md (hunk 1 above)
Comment (azaslavsky) [reply README.md:98]: Done.

**Comment by azaslavsky** [comment]
I've done a `git push --force-with-lease` to update these changes - they should be ready to merge!

**Review by hubertdeng123 — APPROVED** [review]
woohoo! first PR

**Pull request merged**